shardingsphere-scaling/shardingsphere-scaling-core/src/main/java/org/apache/shardingsphere/scaling/core/config/DumperConfiguration.java

```diff
@@ -17,51 +17,25 @@
 
 package org.apache.shardingsphere.scaling.core.config;
 
-import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.Setter;
-import lombok.SneakyThrows;
 import org.apache.shardingsphere.scaling.core.job.position.PositionManager;
 
 import java.util.Map;
-import java.util.Set;
 
 /**
- * Relational database management system configuration.
+ * Dumper configuration.
  */
 @Setter
 @Getter
-@EqualsAndHashCode
-public final class RdbmsConfiguration implements Cloneable {
-    
+public class DumperConfiguration implements Cloneable {
+
     private String dataSourceName;
-    
+
     private DataSourceConfiguration dataSourceConfiguration;
-    
-    private String tableName;
-    
-    private Map<String, Set<String>> shardingColumnsMap;
-    
-    private String primaryKey;
-    
+
     @SuppressWarnings("rawtypes")
     private PositionManager positionManager;
```

Comment:
how about replace SuppressWarnings with generics? or should we extract positionManager to context?